pulsar-metadata/src/main/java/org/apache/pulsar/metadata/cache/impl/MetadataCacheImpl.java

```diff
@@ -268,6 +268,7 @@ public void accept(Notification t) {
         case Modified:
             if (objCache.synchronous().getIfPresent(path) != null) {
                 // Trigger background refresh of the cached item
+                objCache.synchronous().invalidate(path);
```

Comment:
can you please a comment to why this?

Reply:
this change was subsumed by changes committed in #11267. This PR now only has the unit test case.